### PR TITLE
feature(unlock-app): Remove `next-seo`

### PR DESCRIPTION
**Pull request by 0xTxbi**
# Description
This PR introduces the removal of `next-seo` as a dependency, migrating relevant dashboard components to leverage Next.js’s native `generateMetadata`.

# Issues
Fixes #
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/content/event/EventDetails.tsx [hunk 1]
```diff
@@ -193,63 +189,6 @@ export const EventDetails = ({
...
-          },
-          {
-            name: 'fc:frame:image',
-            content: `${config.unlockApp}/og/event/${event.slug}`,
-          },
-          {
-            name: 'fc:frame:post_url',
-            content: `${config.unlockApp}/frames/event?p=${encodeURIComponent(
-              `${config.unlockApp}/frames/event/${event.slug}`
-            )}&s=${encodeURIComponent('{"view":"default"}')}`,
```
Comment (julien51) [inline unlock-app/src/components/content/event/EventDetails.tsx:229]: So I wonder about all of this. Is it gone?

File: unlock-app/src/components/content/subscription/SubscriptionLandingPage.tsx [hunk 2]
```diff
@@ -119,18 +118,6 @@ export const SubscriptionLandingPage = ({
...
-        description="SUBSCRIPTIONS by Unlock Labs is best way for fans to subscribe to writing, podcasts, music, digital art, and creativity onchain with automatically recurring payments."
-        openGraph={{
-          images: [
-            {
-              alt: 'Onchain subscriptions',
-              url: 'https://subscriptions.unlock-protocol.com/',
-            },
-          ],
-        }}
-      />
```
Comment (julien51) [inline unlock-app/src/components/content/subscription/SubscriptionLandingPage.tsx:133]: Same here?

**Review by julien51 — COMMENTED** [review]
Are still having the defaukt OG stuff for every page?

**Comment by 0xTxbi** [comment]
> Are still having the defaukt OG stuff for every page?

yes we do!

File: unlock-app/src/components/content/event/EventDetails.tsx (hunk 1 above)
Comment (0xTxbi) [reply unlock-app/src/components/content/event/EventDetails.tsx:229]: nope!

**Review by julien51 — APPROVED** [review]

**Pull request merged**